Changelog.md

```diff
@@ -6,6 +6,11 @@
 
 ### ✨ New features and improvements
 
+- Add visual indicator on a per-assignment basis for used grace credits (#7226)
+- Improve textviewer rendering speed (#7211)
```

Comment:
Lines 10 through 13 are from a merge

Reply:
This merge is incorrect. These lines were moved under v2.5.2, and should be removed from the unreleased section.

In general, there *shouldn't* be any "lines from a merge". If there is code that appears in a diff on GitHub that you didn't mean to include yourself, that means that there was an error in the merge process that should be resolved.